packages/mui-material-next/CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing
+
+The Material You components are targeted for v6, so they will be developed on the `material-next` package.
+
+The progress for each component will be tracked in a separate GitHub issue. If you wish to contribute to the migration go to a component’s linked issue to see what tasks are missing (see progress tracker [here](https://github.com/mui/material-ui/issues/29345)).
+
+If the issue doesn’t exist, create it, name it `[<ComponentName>][material-next] Add <ComponentName> component`, and assign @DiegoAndai. The issue has to contain (at least) a task for each of the steps below (use the [Slider issue](https://github.com/mui/material-ui/issues/37527) as a template).
+
+## Steps
+
+1. Copy component files from `material` (v5) to `material-next`, including tests, types, and utils. Keep in mind to:
+    - Add component export to `packages/mui-material-next/src/index.ts`
+    - Change imports from `@mui/material` to `@mui/material-next`
+    - If there are utils that don’t exist in `material-next`, add them by copying from `material`
+    - Some utils imported from `../utils` might already exist in `@mui/utils`, if so, use the latter
+2. Migrate component to Typescript. The extension `.d.ts` should be replaced with `.types.ts` (except for `index.d.ts` which won’t be necessary)
+3. Remove deprecated `components` and `componentsProps` props, leaving only `slots` and `slotProps`
+4. Drop support for `ThemeProvider` in favor of `CssVarsProvider`. In practice, this means:
+    - Consuming tokens from `theme.vars` instead of `theme`
+    - In tests, using `CssVarsProvider` and `extendTheme` (both imported from `@mui/material-next/styles`) instead of `ThemeProvider` and `createTheme`, as well as providing the same `CssVarsProvier` and `extendTheme` to `describeConformance`'s `ThemeProvider` and `createTheme` options.
+5. Refactor component to use Base UI hooks if it exists
```

Comment:
```suggestion
```

We can drop this, it can be added later, especially if we aim at releasing only Material You as a change in v6 without other unexpected breaking changes